logseq-dark-dale502.user.css

```diff
@@ -0,0 +1,156 @@
+:root {
```

Comment:
before the `:root`, you should add the front matter to adhere to stylus user.css format and enable auto-intall / auto update 
```
/* ==UserStyle==
@name           Logseq-dark-dale502
@namespace      https://github.com/dale502/logseq-dark-hpx
@version        0.1.5
@description    alternate dark mode for logseq by dale502
@author         _ph / www.hpx1.com / Dale 502
@updateURL      https://raw.githubusercontent.com/dale502/logseq-dark-hpx/master/logseq-dark-dale502.user.css
==/UserStyle== */

@-moz-document domain("logseq.com") {
```
if you want to use  the font Cairo you can add this (this is optionnal) :
```
/* latin */
@font-face {
  font-family: 'Cairo';
  font-style: normal;
  font-weight: 400;
  font-display: swap;
  src: local('Cairo Regular'), local('Cairo-Regular'), url(https://fonts.gstatic.com/s/cairo/v7/SLXGc1nY6HkvalIhTps.woff2) format('woff2');
  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
}
```